### PR TITLE
Various logging fixes

### PR DESCRIPTION
**Pull request by stefanceriu**
* imported the ObjCWrapper and updated the MXLogger. Also added back support for app containers
* updated MXLog.swift from El and fixed/improved its logging format
* updated most app log strings

### CODE REVIEW

File: ElementX/Sources/Other/Logging/MXLog.swift [hunk 1]
```diff
@@ -118,32 +118,57 @@ private var logger: SwiftyBeaver.Type = {
...
+    ///
+    /// A failure is any type of programming error which should never occur in production. In `DEBUG` configuration
+    /// any failure will raise `assertionFailure`
+    ///
+    /// - Parameters:
+    ///     - message: Description of the error without any variables (this is to improve error aggregations by type)
+    ///     - context: Additional context-dependent details about the issue
+    public static func failure(_ message: String,
                                _ file: String = #file,
                                _ function: String = #function,
```
Comment (pixlwave) [inline ElementX/Sources/Other/Logging/MXLog.swift:153]: Not related to this PR, but seeing as we basically never enter these, maybe we should add the parameter names at some stage so its clear which default value you are replacing.
```suggestion
                               file: String = #file,
                               function: String = #function,
```

File: ElementX/Sources/Other/Logging/MXLogger.m [hunk 2]
```diff
@@ -142,7 +143,7 @@ + (void)deleteLogFiles
     }
 }
 
-+ (NSArray<NSString*>*)logFiles
++ (NSArray*)logFiles
```
Comment (pixlwave) [inline ElementX/Sources/Other/Logging/MXLogger.m:146]: How come the array type is removed here?

File: ElementX/Sources/Services/BugReport/BugReportService.swift [hunk 3]
```diff
@@ -106,6 +108,11 @@ class BugReportService: BugReportServiceProtocol {
                 params.append(MultipartFormData(key: "compressed-log", type: .file(url: url)))
             }
         }
+        
+        if let crashEventId = lastCrashEventId {
+            params.append(MultipartFormData(key: "crash_report", type: .text(value: "<https://sentry.tools.element.io/organizations/element/issues/?project=44&query=\(crashEventId)>")))
```
Comment (pixlwave) [inline ElementX/Sources/Services/BugReport/BugReportService.swift:113]: This is clever 🧐

**Review by pixlwave — APPROVED** [review]
LGTM, although I'm not sure I understand why we want to add more objc to the project 🫣

File: ElementX/Sources/Other/Logging/MXLog.swift [hunk 4]
```diff
@@ -152,24 +177,27 @@ private var logger: SwiftyBeaver.Type = {
...
             return
         }
         
+        logger.removeAllDestinations()
+        
         let consoleDestination = ConsoleDestination()
         consoleDestination.useNSLog = true
         consoleDestination.asynchronously = false
-        consoleDestination.format = "$M"
+        consoleDestination.format = "$C$N.$F:$l $M $X$c" // Format is `Color Message Context`, see https://docs.swiftybeaver.com/article/20-custom-format
```
[the remark this answers is not in the document]
Comment (stefanceriu) [reply ElementX/Sources/Other/Logging/MXLog.swift:195]: Yup, definitely 👍

File: ElementX/Sources/Other/Logging/MXLogger.m (hunk 2 above)
Comment (stefanceriu) [reply ElementX/Sources/Other/Logging/MXLogger.m:146]: Seems that Ismail added that when migrating it over from Element. It got overriden when I copied everything over again

**Pull request merged**